source/playgrounds.txt

```diff
@@ -30,12 +30,35 @@ files with the ``.mongodb`` extension as playgrounds.
 
 .. include:: /includes/admonitions/note-playgrounds.rst
 
-|vsce| splits your Playground window to display the results of your 
-Playground in |json| format in the right-side pane labeled ``Playground 
-Results.json``. If you disabled split-view, |vsce| displays the 
-Playground results in |json| format in a new tab to the right of 
-your Playground. You cannot edit the results in the results pane. 
-You can save the results to a file. 
+|vsce| splits your Playground window to display your
+:guilabel:`Playground Result` in the right-side pane.
+
+If |vsce| returns these results in |json| format, you may edit these
+results. If the result returns |json|\-formatted content, |vsce|
+displays it with |json| syntax highlighting and includes an
```

Comment:
Delete this sentence: `If |vsce| returns these results in |json| format, you may edit these results.`

Replace `If the result returns |json|\-formatted content` with `If the playground returns |json|\-formatted content`

Reply:
This was rewritten before this change.